syntax = "proto2";

message AuthenticationRequest{
	required string username = 1;
	required string password = 2;
}
